pyfar/dsp/regularization.py

```diff
@@ -0,0 +1,408 @@
+"""
+Documentation for regularization class.
+"""
+
+import pyfar as pf
+from pyfar.dsp.dsp import _cross_fade
+import numpy as np
+import numbers
+
+class RegularizedSpectrumInversion():
+    r"""Class for frequency-dependent regularized inversion.
+
+    Regularized inversion limits the gain applied by an inverse filter.
+    This can be useful to avoid extreme amplification when inverting a signal
+    of limited bandwidth or containing notches in the spectrum.
+
+    The inverse is computed as [#]_:
+
+    .. math::
+        :label: regularized_inversion
+
+        S^{-1}(f) = \frac{S^*(f)}{S^*(f)S(f) + \beta |\epsilon(f)|^2} D(f)
+
+    with :math:`f` being the frequency, :math:`S(f)` the spectrum of the signal
+    to be inverted, :math:`(\cdot)^*` the complex conjugate,
```

Comment:
```suggestion
    to be inverted, :math:`*` the complex conjugate,
```
looks somehow odd in my opinion